### PR TITLE
Add median to prod functions

### LINKED ISSUE #173: median  is missing from the output of `prod_gd_compute_pip_stats()`

### LINKED ISSUE #174: median should be adjusted for PPP in `prod_md_compute_pip_stats()`

### PR DESCRIPTION
**Pull request by Aeilert**
Add median to production functions, so that it can be adjusted if the user changes the PPP.

Closes #173. 
Closes #174.

### CODE REVIEW

File: R/prod_gd_compute_pip_stats.R [hunk 1]
```diff
@@ -21,13 +21,23 @@ prod_gd_compute_pip_stats <- function(welfare,
                                       povline,
                                       population,
                                       requested_mean,
+                                      svy_median_lcu,
+                                      svy_median_ppp,
                                       popshare = NULL,
-                                      default_ppp = NULL,
+                                      default_ppp = 1,
```
Comment (tonyfujs) [inline R/prod_gd_compute_pip_stats.R:27]: I'm not sure we should have a default ppp value of 1 here... Expected behavior is default_ppp is not provided would be to fail explicitly no? A default value of 1 would instead give incorrect results...

**Review by tonyfujs — COMMENTED** [review]
Hey @Aeilert 
I left a comment. Let me know what you think. Thanks!

File: R/prod_gd_compute_pip_stats.R (hunk 1 above)
Comment (Aeilert) [reply R/prod_gd_compute_pip_stats.R:27]: Hey @tonyfujs, 

Yes, the below functions will fail if `default_ppp` is `NULL` . If that is the expected behavior I would suggest that `default_ppp` is set wo/ any default argument. Wouldn't that give the most explicit error from R?

Comment (Aeilert) [reply R/prod_gd_compute_pip_stats.R:27]: To illustrate: This won't work if default ppp is NULL

```r
  # Adjust mean if different PPP value is provided
  if (!is.null(ppp)) {
    requested_mean <- requested_mean * default_ppp / ppp
  } else {
    ppp <- default_ppp
  }
```

Comment (tonyfujs) [reply R/prod_gd_compute_pip_stats.R:27]: Thanks. Yeah, leaving it empty probably makes more sense than the NULL default.

Comment (Aeilert) [reply R/prod_gd_compute_pip_stats.R:27]: Will fix.

Comment (Aeilert) [reply R/prod_gd_compute_pip_stats.R]: Hi @tonyfujs, I changed `default_ppp` to empty for the prod functions.

Additionally I attempted to add some consistency in how the `default_ppp` is set for the regular gd_* and  md_* functions. Currently `md_compute_pip_stats` had `default_ppp` set to `1` as a default argument, while `gd_compute_pip_stats()` had it set to `NULL` . I set both to 1 for the main functions, as well as setting it to empty for the lower level `_lb` and `_lq` GD functions (same logical as above, NULL doesn't really make sense for the ppp adjustment.)

**Pull request merged**